codebases/index.html

```diff
@@ -0,0 +1,88 @@
+---
+layout: codebase
+title: Codebases we work with
+---
+
+<div class="cb-section">
+  <div>
+    <h1 class="title">Codebases we work with</h1>
+    <h4 class="subtitle"><p>The Foundation for Public Code helps to effectively scale codebases. We do this by supporting ambitious communities with <a href="/codebase-stewardship/">codebase stewardship</a>.</p>
+    <p>Codebases are organized by their <a href="https://about.publiccode.net/activities/codebase-stewardship/lifecycle.html">codebase maturity</a>.</p></h4>
+  </div>
+</div>
+
+<div class="cb-maturity-index-header cb-maturity-s1">
+  <h1 class="title">Stage 1: <strong>Assessment</strong></h1>
+  <p><strong>In this phase, codebase communities and the Foundation for Public Code decide if:</strong></p>
+  <ul>
+    <li>the codebase is a good fit for Foundation stewardship</li>
```

Comment:
Should "Foundation" in "Foundation stewardship" be written in full like "Foundation for Public Code stewardship"? (Applies to the use of "Foundation" four more times on this page.)

Reply:
I've done this across the website now, except for the job ads.